cmd/kubeadm/app/util/apiclient/wait_test.go

```diff
@@ -24,13 +24,16 @@ import (
 	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
 )
 
+// TestGetControlPlaneComponents tests the getControlPlaneComponents function
 func TestGetControlPlaneComponents(t *testing.T) {
```

Comment:
The function name is enough to understand, so we may not need to explain it. Other comments are not necessary, the code is self-explanatory.

Reply:
Thanks for your willingness to improve the code. Welcome to the community.

Reply:
No problem, are there any areas where these kind of changes could be used?